packages/eas-cli/src/commands/worker/deploy.ts

```diff
@@ -253,21 +253,28 @@ export default class WorkerDeploy extends EasCommand {
     let progress = ora('Preparing project').start();
 
     try {
-      const manifest = await WorkerAssets.createManifestAsync(
+      const manifestResult = await WorkerAssets.createManifestAsync(
         {
           environment: flags.environment,
           projectDir,
           projectId,
         },
         graphqlClient
       );
+      if (manifestResult.conflictingVariableNames?.length) {
+        Log.warn(
+          '> The following environment variables were loaded both from local .env files as well as EAS environment variables, ' +
+            ' and will be set to the EAS environment variable values instead: ' +
```

Comment:
```suggestion
          '> The following environment variables were present in local .env files as well as EAS environment variables, ' +
            ' in case of conflict, the EAS environment variable values will be used: ' +
```

Perhaps this wording is a bit clearer, to avoid saying "loaded both from" and "will be set" which is an unnecessary implementation detail. Not a big deal though.